src/components/suggest/suggest_item.tsx

```diff
@@ -123,14 +150,20 @@ export const EuiSuggestItem: FunctionComponent<EuiSuggestItemProps> = ({
 
   let colorClass = '';
 
-  const labelDisplayClass = classNames(
+  const labelClassNames = classNames(
     'euiSuggestItem__label',
     labelDisplayToClassMap[labelDisplay],
```

Comment:
Above here, we do a "calculated" label display first based on if the description exists and then if it does, use  the `labelDisplay`. And use that to find teh right class name so it's only displayed once.

```suggestion
  const labelDisplayCalculated = !description ? 'expand' : labelDisplay;
  
  const labelClassNames = classNames(
    'euiSuggestItem__label',
    labelDisplayToClassMap[labelDisplayCalculated],
```